packages/ui/cart/drawer/Upsell.tsx

```diff
@@ -51,6 +53,7 @@ export function Upsell({ item }: UpsellProps) {
             borderWidth="1px"
             borderColor="brand.200"
             borderStyle="solid"
+            onClick={onClose}
```

Comment:
I'm curious why this is necessary. Does the cart drawer remain open after navigation?

Reply:
Yes, you could click the link and the cart would stay open. This closes it.